### PR TITLE
[stable/sentry] Fix wrong hierarchy of imagePullSecrets and imagePull…

### PR DESCRIPTION
**Pull request by Antiarchitect**
…Policy fields in README.

Signed-off-by: Andrey Voronkov <avoronkov@enapter.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/sentry/README.md [hunk 1]
```diff
@@ -64,8 +64,8 @@ Parameter                                            | Description
 :--------------------------------------------------- | :--------------------------------------------------------------------------------------------------------- | :---------------------------------------------------
 `image.repository`                                   | Sentry image                                                                                               | `library/sentry`
 `image.tag`                                          | Sentry image tag                                                                                           | `9.1.2`
-`imagePullPolicy`                                    | Image pull policy                                                                                          | `IfNotPresent`
-`imagePullSecrets`                                   | Specify image pull secrets                                                                                 | `[]`
+`image.imagePullPolicy`                              | Image pull policy                                                                                          | `IfNotPresent`
```
Comment (okgolove) [inline stable/sentry/README.md]:
```suggestion
`image.PullPolicy`                              | Image pull policy                                                                                          | `IfNotPresent`
```

https://github.com/helm/charts/blob/master/stable/sentry/values.yaml#L7

**Review by okgolove — CHANGES_REQUESTED** [review]
Hi @Antiarchitect. Thank you for the PR.

Nice catch. Could you also fix the pull policy variable name?

**Comment by Antiarchitect** [comment]
@okgolove 
Should it be `image.pullPolicy` and `image.pullSecrets` ?

**Comment by okgolove** [comment]
@Antiarchitect I think we shouldn't make breaking changes, so variables name will be the same.

**Comment by Antiarchitect** [comment]
So nothing to do from my side?

**Comment by okgolove** [comment]
I meant changing in the documentation, there is a value **pullPolicy** in the values.yaml
https://github.com/helm/charts/blob/master/stable/sentry/values.yaml#L7

And it is called **imagePullPolicy** in the doc. Could you change the name in the doc? Not change the value in the file.
https://github.com/helm/charts/pull/18244/files#diff-32a7042994b0ce85310e575475d30b81R67

**Comment by okgolove** [comment]
I meant changing in the documentation, there is a value **pullPolicy** in the values.yaml
https://github.com/helm/charts/blob/master/stable/sentry/values.yaml#L7

And it is called **imagePullPolicy** in the doc. Could you change the name in the doc? Not change the value in the file.
https://github.com/helm/charts/pull/18244/files#diff-32a7042994b0ce85310e575475d30b81R67

**Comment by Antiarchitect** [comment]
@okgolove Now I understand :) Fixed.

**Pull request merged**